File: ticketProject6-RMI/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/BoughtTicketResponse.java
package network.objectprotocol;

import network.dto.ArtisticRepresentationDTO;

public class BoughtTicketResponse implements Response {
    private ArtisticRepresentationDTO ar;

    public BoughtTicketResponse(ArtisticRepresentationDTO ar) {
        this.ar = ar;
    }

    public ArtisticRepresentationDTO getAR() {
        return ar;
    }
}

File: ticketProject7-CrossPlatform/TS-crossp/networking/response/BoughtTicketResponse.cs
using System;
namespace networking.protocol
{

	using ArtisticRepresentationDTO = networking.dto.ArtisticRepresentationDTO;

	[Serializable]
	public class BoughtTicketResponse : UpdateResponse
	{
		private ArtisticRepresentationDTO artisticRepresentationDTO;

		public BoughtTicketResponse(ArtisticRepresentationDTO ar)
		{
			this.artisticRepresentationDTO = ar;
		}

		public virtual ArtisticRepresentationDTO ArtisticRepresentation
		{
			get
			{
				return artisticRepresentationDTO;
			}
		}
	}

}